cmd/influx/pkg.go

```diff
@@ -91,6 +103,83 @@ func pkgApply(orgID, path *string, hasColor, hasTableBorders *bool) func(*cobra.
 	}
 }
 
+func pkgCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Create a reusable pkg to create resources in a declarative manner",
+	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
```

Comment:
intentionally panicing here, it should never reach here, and if it does, we have bigger problems 🤷‍♂